contract-map.json

```diff
@@ -356,6 +356,27 @@
     "symbol": "ZERO",
     "decimals": 18
   },
+  "0x74C9F2B00581F1b11Aa7Ff05aa9f608B7389de67": {
+    "name": "Bridged USDC (Stargate)",
+    "logo": "usdc.png",
+    "erc20": true,
+    "symbol": "USDC.e",
+    "decimals": 6
+  },
+  "0xAF368c91793cb22739386DFCBbB2f1A9E4bcBEBf": {
+    "name": "USDT (Stargate)",
```

Comment:
Let's just leave USDT as name (as the contract says)

Reply:
Done!